tasks/packages-apt.yml

```diff
@@ -2,15 +2,20 @@
 
 - name: Update cache
   apt:
-    update_cache: yes
-    cache_valid_time: 43200 #12 hours
+    update_cache: true
+    cache_valid_time: 43200   # 12 hours
 
 - name: Install git
   apt:
     name: git
     state: present
   when: caddy_features is search('git')
 
+- name: Install procps
```

Comment:
Why is this needed?

Reply:
The Debian 10 docker image doesn't come with this package (which provides the `/bin/kill` command used in the systemd service file for reloading caddy). I thought about just ensuring it was installed for the test but if it's genuinely needed for the role then it seemed better to just put it here.

Reply:
Perhaps I should also ensure the package which provides this on other distros is installed too though?

Reply:
Ok, then please add a comment why procps is needed. It's weird this is needed though, I thought `kill` is a command that is already installed on pretty much every distro.

Reply:
It's a builtin in bash so you can use it without this package being installed (assuming you are in  bash) but the systemd service needs the actual binary.

I was also surprised it's not included tbh.

Reply:
Ah, that makes more sense :) Thanks for clarifying. You can decide regarding the other distros

Reply:
On RedHat-based distros & Arch this is provided by `util-linux`. I think it's reasonable to assume that is always installed so I'm going to leave this as apt-specific.